nservicebus/pipeline/customizing-v6.md

```diff
@@ -50,6 +50,8 @@ The following lists describe some of the common stages which you can build your
 
 ### Outgoing Pipeline Stages
 * Operation specific processing: There is a dedicated stage for each bus operation (e.g. Send, Publish, Subscribe, ...). Behaviors can use one of the following contexts: `IOutgoingSendContext`, `IOutgoingPublishContext`, `IOutgoingReplyContext`, `ISubscribeContext`, `IUnsubscribeContext`.
+* Audit message processing: In here behaviors have access to audited message and audit address. Behaviors should use `IAuditContext` to enlist in this stage.
```

Comment:
IAuditContext has property message with a comment: The message to be audited. Do your question refer to:
a. The message is not audted yet. 
b. That this is actually the audit message, not the original message that my comment is suggesting?